### DISCUSSION TITLE
localStorage...에 저장한다?

### DISCUSSION THREAD

**Discussion by parkjooyoung0608** [EasyAndBeauty/DailyReceipt-FE-WEB#42] [Q&A] [answered]
fetch로 요청 받는거랑 localStorage에 저장하는거랑 무슨차이가 있는걸까요...?!

**Comment by IGhost-P** [comment, marked as answer]
fetch는 인터넷을 통해 외부 네트워크와 통신을 합니다, 그렇게 되면 외부에 있는 DB나 백엔드 서버와 통신이 가능합니다
localStorage는 브라우저에서 해당하는 url에 저장할수있는 작은 DB 같은 개념입니다! local은 인터넷 연결이 없어도 사용이 가능하다는 장점이 있습니다! 하지만 쿠키 삭제 & 브라우저 초기화시 사라진다는 단점이 있습니다

**Reply by parkjooyoung0608** [reply]
오!! 그런 차이점이 있었군요..! 친절하게 설명해줘서 고마워요 댄 :)
